settings.gradle.kts

```diff
@@ -52,6 +52,7 @@ include(":graphql-kotlin-spring-server")
 
 // Executions
 include(":graphql-kotlin-transaction-batcher")
+include(":graphql-kotlin-transaction-batcher-instrumentation-level")
```

Comment:
this is a very long artifact name - we can drop the `-level` but wondering whether something like `graphql-kotlin-batch-instrumentation` is sufficient (probably it would make sense to keep naming consistent so maybe `transaction-batcher-instrumentation` or do we rename the `transaction-batcher` to something shorter?